src/api/dataclasses/cf_response_objects.py

```diff
@@ -0,0 +1,55 @@
+"""Dataclasses for the CurseForge API to process response data.
+"""
+from dataclasses import dataclass
+from typing import Optional
+
+
+@dataclass
+# pylint: disable=too-many-instance-attributes
+class CFGetModData:
+    """
+    Metadata for CurseForge Mod
+    """
+    id: int
+    # pylint: disable=invalid-name
+    gameId: int
+    name: str
+    slug: str
+    links: dict
+    summary: str
+    status: int
+    downloadCount: int
+    primaryCategoryId: int
+    classId: int
+    authors: list[dict]
+    mainFileId: int
+    latestFiles: list[dict]
+    latestFileIndexes: list[dict]
+    latestEarlyAccessFilesIndexes: list[dict]
+    dateCreated: str
+    dateModified: str
+    allowModDistribution: bool
+    isAvailable: bool
+
```

Comment:
`# pylint: enable=invalid-name`

Reply:
This would just trigger the `invalid-name` rule on all of the fields in this dataclass, which we don't want because it follows the CurseForge API's naming conventions. I'll add some documentation to clarify this dataclass' purpose.